INSTALL.md

```diff
@@ -101,6 +101,12 @@ Install Sheer’s Python requirements:
 pip install -r ~/path/to/sheer/requirements.txt
 ```
 
+### Server Setup
+Install required dependencies
+```bash
+pip install -r requirements/base.txt
```

Comment:
Is this something that would make sense to have in the install step in `./setup.sh`?